### PR TITLE
fix: calling infinite vibration (AR-2433) (AR-2363)

### PR DESCRIPTION
**Pull request by alexandreferris**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2433" title="AR-2433" target="_blank">BugAR-2433</a>  Call kept vibrating forever on background & Wrong info about caller
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [X] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [X] contains a reference JIRA issue number like `SQPIT-764`
  - [X] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [X] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

When receiving an incoming call notification, sometimes the vibration would continue infinitely.

### Solutions

_Briefly describe the solutions you have implemented for the issues explained above._

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-986/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4426751)(4426751)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4426751)(4426751)-dev-release.apk)

**Review by MohamadJaara — APPROVED** [review]
📳

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-986/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4428469)(4428469)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4428469)(4428469)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/notification/CallNotificationManager.kt [hunk 1]
```diff
@@ -57,13 +56,15 @@ class CallNotificationManager @Inject constructor(private val context: Context)
     private fun createIncomingCallsNotificationChannel() {
         val audioAttributes = AudioAttributes.Builder()
             .setContentType(AudioAttributes.CONTENT_TYPE_MUSIC)
-            .setUsage(AudioAttributes.USAGE_MEDIA)
+            .setUsage(AudioAttributes.USAGE_NOTIFICATION_RINGTONE)
```
Comment (yamilmedina) [inline app/src/main/kotlin/com/wire/android/notification/CallNotificationManager.kt]: @alexandreferris It seems this did the trick for me 🤯 Now the vibration is stopping 👌

**Review by yamilmedina — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-986/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4436555)(4436555)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4436555)(4436555)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-986/6/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4436688)(4436688)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4436688)(4436688)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [7](https://10.10.124.134/job/wire-android-reloaded/job/PR-986/7/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4436862)(4436862)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4436862)(4436862)-dev-release.apk)

**Pull request merged**